docker-compose/cfg/configure.py

```diff
@@ -0,0 +1,596 @@
+#!/bin/env python3
+import time
+import json
+from jinja2 import Template
+import npyscreen
+import weakref
+
+
+def load_default_config():
+    # Read defaults
+    with open("defaults") as defaults_file:
+        defaults = json.load(defaults_file)
+    return defaults
+
+
+def load_current_config():
+    # Read current config
+    with open("/docker-compose/current_config") as current_config:
+        config = json.load(current_config)
+    return config
+
+
+##
+# Help text in a box at the bottom of the screen
+class HelpBox(npyscreen.BoxTitle):
+    def display_help_message(self, field):
+        if field == "Wallet Address:":
+            self.set_values(
+                [
+                    "Your monero wallet address for receiving mining reqards",
+                    "",
+                    "Note: You have to use a primary wallet address for mining",
+                    "      Subaddresses and integrated addresses are not supported!",
+                ]
+            )
+        elif field == "Configure Monero Node":
+            self.set_values(
+                [
+                    "Configure and run a Monero Node",
+                    "",
+                    "Note: You must either configure a local node or specify a public node",
+                ]
+            )
+        elif field == "Configure XMRig CPU Miner":
+            self.set_values(
+                [
+                    "Configure and run an XMRig CPU Miner",
+                    "",
+                    "Note: You must either configure am XMRig CPU Miner or expose the P2Pool",
+                    "      stratum port and connect an external miner, or both",
+                ]
+            )
+        elif field == "P2Pool Sidechain:":
+            self.set_values(
+                [
+                    "Which P2Pool sidechain to mine on",
+                    "    use main for faster miners",
+                    "    use mini for slower miners",
+                ]
+            )
+        elif field == "Expose Stratum Port":
+            self.set_values(
+                [
+                    "Expose the P2Pool stratum port to your network so external miners can connect",
```

Comment:
This line doesn't fit in the window when I run it, it stops at "miners can".

Reply:
Thanks for the thorough review.  I couldnt get the npyscreen resize to work perfectly so I set a fixed windows size for now with intent to work on the resize as a future enhancement.  I'll adjust the text line length to avoid truncation for now and push the fix a little later today.

Reply:
fixed that (and one other case) plus minor formatting adjustments to other help text.